scripts/asm-installer/asm_vm

```diff
@@ -1110,6 +1110,22 @@ this project, run:
   --project="${PROJECT_ID}"
 EOF
     fi
+
+    local INSTANCE_TEMPLATE_REQ=$(retry 2 run gcloud compute instance-templates \
+      describe "${SOURCE_INSTANCE_TEMPLATE}" \
+      --format=json \
+      --project="${PROJECT_ID}")
+    local source_image_path="$(echo "${INSTANCE_TEMPLATE_REQ}" \
+    | jq '.properties.disks[].initializeParams.sourceImage')"
+
+    # check whether it is a  public gcp image
+    if [[ "$source_image_path" == *projects* && "$source_image_path" == *global/images/* ]]; then
+      if [[ "$source_image_path" != *debian-9* ]] && [[ "$source_image_path" != *debian-10* ]]; then
+        fatal "found not support image, the image path is $source_image_path"
```

Comment:
@jasonwzm are there any test files that should be used for adding tests to cover this change?

Reply:
Yes. There are two test files in /tests folder that cover VM.

Reply:
Please also add test in the same PR.